### PR TITLE
Upgrade to Xcode 8.1, fix warnings

### PR DESCRIPTION
**Pull request by skyline75489**

### CODE REVIEW

File: SwiftyJSON.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -757,7 +757,7 @@
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = YES;
 				BITCODE_GENERATION_MODE = bitcode;
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "";
```
Comment (zhigang1992) [inline SwiftyJSON.xcodeproj/project.pbxproj]: Can you help revert this changes?
I believe we wanna keep them as "iPhone Developer". 😄

**Comment by skyline75489** [comment]
Any specific reason for this？Cuz Xcode gives warnings about 'iPhone Developer'

**Comment by zhigang1992** [comment]
It was added here

https://github.com/SwiftyJSON/SwiftyJSON/pull/682/files

maybe @mokagio have more context here.

**Comment by ldiqual** [comment]
@mokagio Bump. Can you give more context on the changes of your PR and why it creates warnings in 8.1? Thank you!

**Comment by wongzigii** [comment]
Closing. As of Xcode 8.2 recommend settings in #764 
Thanks everyone. 🙏
